src/vmm/src/dumbo/tcp/connection.rs

```diff
@@ -100,23 +100,26 @@ pub enum RecvError {
 }
 
 /// Describes errors which may occur when a connection attempts to write a segment.
-#[derive(Debug, PartialEq, Eq, derive_more::From)]
+/// Needs `rustfmt::skip` to make multiline comments work
+#[rustfmt::skip]
+#[derive(Debug, PartialEq, Eq, thiserror::Error, displaydoc::Display)]
 pub enum WriteNextError {
     /// The connection cannot write the segment because it has been previously reset.
     ConnectionReset,
     /// The write sends additional data after a `FIN` has been transmitted.
     DataAfterFin,
-    /// The remaining MSS (which can be reduced by IP and/or TCP options) is not large enough to
-    /// write the segment.
+    /** The remaining MSS (which can be reduced by IP and/or TCP options) is not large enough to \
+    write the segment. */
```

Comment:
nit: Ah, neat! Can we use this instead of the `#[doc(...)]` directives we use sometimes for long display strings?

Reply:
I guess so. It still needs `#[rustfmt::skip]` though.

Reply:
We can open an issue to replace `#[doc = ]` to `/** */`.